.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - DOCKER_VERSION=1.12.6 ENABLE_SWARM=1 UPLOAD_SONATYPE=1
     - DOCKER_VERSION=1.13.1 ENABLE_SWARM=1
     - DOCKER_VERSION=17.03.0~ce ENABLE_SWARM=1
+    - DOCKER_VERSION=17.04.0~ce ENABLE_SWARM=1
```

Comment:
To keep the build matrix limited, I'm fine with replacing 17.03 with 17.04. Thoughts @mattnworb?